### DISCUSSION TITLE
Radio Thermostat Company of America (RTC) CT101 Thermostat Fan mode missing

### DISCUSSION THREAD

**Discussion by cookemmm** [zwave-js/zwave-js-ui#2802] [Q&A]
I am moving from openzwave to zwavej22mqtt for use with domoticz which uses the Hass devices. Everything seems good except there is no device to control the fan mode. I was trying to follow the docs to add it myself but can't seem to figure it out and thought maybe someone with more experience could help me out.

**Comment by AlCalzone** [comment]
This is probably mqtt related so you might have more luck in the zwave-js-ui repository.

CC @robertsLando

**Comment by robertsLando** [comment]
@cookemmm check out this: https://zwave-js.github.io/zwave-js-ui/#/homeassistant/homeassistant-mqtt?id=custom-components

**Reply by cookemmm** [reply]
Thanks I tried following that documentation but did not have much success.....I will try the zwave-js-ui repository as suggested.

**Reply by robertsLando** [reply]
> I will try the zwave-js-ui repository as suggested.

🤔  aren't you already using zwave-js-ui?

**Reply by AlCalzone** [reply]
I meant asking there 🙃
